x-pack/auditbeat/module/system/packages/_meta/fields.yml.disabled

```diff
@@ -0,0 +1,50 @@
+- name: packages
```

Comment:
What's the purpose of the file `fields.yml.disabled`? I've seen the disabled config files, but that's the first time I see a disabled fields file. Leftover from some tests around field definitions?

Reply:
Actually I just noticed docs.asciidoc.disabled above, as well. I'm intrigued :-)

Reply:
This is part of disabling the `packages` metricset (https://github.com/elastic/beats/pull/9495). Renaming the files prevents them from being picked up by the various build scripts.